### PR TITLE
Create `FeedbackBatchAnalysisOrchestrationService`

### PR DESCRIPTION
**Pull request by Ilxom-22**

### CODE REVIEW

**Review by isxoqovxasanboy — APPROVED** [review]

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/FeedbackAnalysisWorkflows/EventHandlers/AnalyzeFeedbackEventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using Feedback.Analyzer.Application.FeedbackAnalysisWorkflows.Events;
+using Feedback.Analyzer.Application.FeedbackAnalysisWorkflows.Services;
+using Feedback.Analyzer.Domain.Common.Events;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Feedback.Analyzer.Infrastructure.FeedbackAnalysisWorkflows.EventHandlers;
+
+public class AnalyzeFeedbackEventHandler(IServiceScopeFactory serviceScopeFactory) : IEventHandler<AnalyzeFeedbackEvent>
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/FeedbackAnalysisWorkflows/EventHandlers/AnalyzeFeedbackEventHandler.cs:8]: Add docs

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/FeedbackAnalysisWorkflows/EventHandlers/AnalyzeWorkflowFeedbacksEventHandler.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+using Feedback.Analyzer.Application.FeedbackAnalysisWorkflows.Events;
+using Feedback.Analyzer.Application.FeedbackAnalysisWorkflows.Services;
+using Feedback.Analyzer.Domain.Common.Events;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Feedback.Analyzer.Infrastructure.FeedbackAnalysisWorkflows.EventHandlers;
+
+public class AnalyzeWorkflowFeedbacksEventHandler(IServiceScopeFactory serviceScopeFactory) : IEventHandler<AnalyzeWorkflowFeedbacksEvent>
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/FeedbackAnalysisWorkflows/EventHandlers/AnalyzeWorkflowFeedbacksEventHandler.cs:8]: Add docs

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/Repositories/EntityRepositoryBase.cs [hunk 3]
```diff
@@ -232,4 +234,30 @@ private string AddTypePrefix(CacheModel model)
...
+    {
+        var entities = DbContext.Set<TEntity>().AsQueryable();
+
+        if (batchUpdatePredicate is not null)
+            entities = entities.Where(batchUpdatePredicate);
+
+        return await entities.ExecuteUpdateAsync(
+            setPropertyCalls,
+            // categoryPropertySelector => categoryPropertySelector.MapToSetPropertyCalls(setPropertyExecutors),
+            cancellationToken
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/Repositories/EntityRepositoryBase.cs:260]: Remove commented codes

**Review by sulton-max — CHANGES_REQUESTED** [review]

**Review by botirjonabdumannonovv — DISMISSED** [review]

**Review by sulton-max — DISMISSED** [review]

**Review by sulton-max — APPROVED** [review]
Good job

**Review by rashidovasadbek — APPROVED** [review]

**Pull request merged**